setup.py

```diff
@@ -31,7 +31,7 @@
     requirements.append("dataclasses")
 
 if not on_readthedocs:
-    requirements.append("tensorflow-probability>=0.9")
+    requirements.append("tensorflow-probability==0.9")
```

Comment:
does this allow for e.g. 0.9.1?

Reply:
No. You can use `~=` for that though.

Reply:
that'd be good

Reply:
```suggestion
    requirements.append("tensorflow-probability~=0.9")
```

Reply:
There is no 0.9.1 and there will not be such thing :)